core/src/main/java/io/questdb/cairo/TableReaderMetadata.java

```diff
@@ -257,4 +220,48 @@ public TableReaderMetadata deferredInit(Path path, int expectedVersion) {
         }
         return this;
     }
+
+    public void dumpTo(MemoryMA mem) {
+        // Since _meta files are immutable and get updated with a single atomic rename
+        // operation replacing the old file with the new one, it's ok to clone the metadata
+        // by copying metaMem's contents. Even if _meta file was already replaced, the file
+        // should be still kept on disk until inode's ref counter is above zero.
+        long len = ff.length(metaMem.getFd());
```

Comment:
this is unfortunate to start here, but we should start dealing with error codes returned from `length()` calls

Reply:
ok removed `ff` from here